src/utils/validateName.ts

```diff
@@ -12,7 +12,8 @@ const PROFILE_NAME_REGEX = /^[\w\-\s]+$/;
 const URL_REGEX = /^((?:(?:http?|ftp)[s]*:\/\/)?[a-z0-9-%\/\&=?\.]+\.[a-z]{2,4}\/?([^\s<>\#%"\,\{\}\\|\\\^\[\]`]+)?)/;
 const WHITE_SPACE_REGEX = /^\S*$/;
 const NUM_REGEX = /^[1-9](\d?|\d+)$/;
-// prettier-ignore
+const SSH_REGEX = /ssh-rsa AAAA[0-9A-Za-z+/]+[=]{0,3}( [^@]+@[^@]+)?/;
```

Comment:
this will work only against `ssh-rsa` keys only, and there are some other types like `ssh-ed25519` that won't be working

Reply:
done.